server/lyft/gateway/autoplan_builder.go

```diff
@@ -98,6 +100,15 @@ func (r *AutoplanValidator) isValid(ctx context.Context, logger logging.Logger,
 		}
 		return false, nil
 	}
+
+	// WorkflowModeType is per repo so if the first ProjectCommand has PlatformWorkflowMode enabled, it's enabled for all projects in the repo
+	// So, we set atlantis/apply status to success to allow PRs to merge to master
+	// TODO: Remove this after we remove the required atlantis/apply status check
+	if projectCmds[0].WorkflowModeType == valid.PlatformWorkflowMode {
```

Comment:
should we instead perform this check + update for each project individually instead of check for one and handle them all?

Reply:
Our current setup enables platform mode for the whole orchestration config for a project and the config generator assigns that to all the roots in that project when generating the atlantis.yaml file. So, if platform mode is enabled for  the any of the roots, we can safely assume that platform mode is enabled on this project.